lib/screens/signup/signup.dart

```diff
@@ -34,7 +34,15 @@ class SignupScreen extends StatelessWidget {
           if (state is SignupError) {
             Scaffold.of(context).showSnackBar(SnackBar(content: Text(state.message)));
           } else if (state is SignupSuccess) {
-            // TODO
+            Navigator.pushReplacement(
```

Comment:
Add a snackbar here telling the user that signup was successful.

Reply:
done